pages/toolkits/community-toolkit-template.mdx

```diff
@@ -1,3 +1,3 @@
-# Arcade <toolkit-name> Toolkit
```

Comment:
Do you not want some sort of indication for placeholders?

Reply:
I do

Reply:
what would you suggest? The prod build is failing because of the previous placeholder <>

Reply:
Does escaping the carats work? `\<`

Reply:
`&lt` and `&gt` would work, but I will just opt for UPPER-CASE as an indication for placeholder.